lib/tasks/webpacker.rake

```diff
@@ -1,13 +1,13 @@
 tasks = {
-  'webpacker:install' => 'Installs and setup webpack with yarn',
-  'webpacker:compile' => 'Compiles webpack bundles based on environment',
-  'webpacker:install:react' => 'Installs and setup example react component',
-  'webpacker:install:vue' => 'Installs and setup example vue component',
-  'webpacker:install:angular' => 'Installs and setup example angular2 component'
+  "webpacker:install"         => "Installs and setup webpack with yarn",
+  "webpacker:compile"         => "Compiles webpack bundles based on environment",
+  "webpacker:install:react"   => "Installs and setup example react component",
+  "webpacker:install:vue"     => "Installs and setup example vue component",
+  "webpacker:install:angular" => "Installs and setup example angular2 component"
 }.freeze
 
-desc 'Lists available tasks under webpacker'
+desc "Lists available tasks under webpacker"
 task :webpacker do
-  puts 'Available webpacker tasks are:'
+  puts "Available webpacker tasks are:"
```

Comment:
Double quotes are used all over the rails's repositories nowadays, also [github's rubocop rules](https://github.com/github/rubocop-github/) is using double quotes too https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#strings so I think we should keep the rule as is.